timer.py

```diff
@@ -6,20 +6,27 @@ def __init__(self):
         # set the font and the starting ticks for calculating timer
         self.font = pygame.freetype.SysFont(None, 34) #34 font size
         self.font.origin=True
-        self.start_ticks =  pygame.time.get_ticks() 
+        self.start_ticks =  0
+        self.running = False
 
+    def start_timer(self):
+        if not self.running:
+            self.start_ticks =  pygame.time.get_ticks() 
+            self.running = True
 
-    def update_timer(self, clock, display):
+    def is_running(self):
```

Comment:
This is just a nit, but I'm not sure this function is needed, I feel like just calling the running variable directly would be fine and self documenting enough.

Reply:
I had it initially to just call the running variable, but then thought back to my CRR class discussion of OO programming and added this abstraction.  Not sure what the best practice is for Python/this situation is actually....